azure-pipelines.yml

```diff
@@ -44,14 +44,20 @@ steps:
   displayName: 'run unit and integration tests'
 
 - script: |
-    ./node_modules/.bin/webdriver-manager update --versions.chrome=83.0.4103.39
+
+    if not exist "C:\Program Files (x86)\Google" mkdir "C:\Program Files (x86)\Google"
+    if not exist "C:\Program Files (x86)\Google\Chrome" mkdir "C:\Program Files (x86)\Google\Chrome"
+    mklink /D "C:\Program Files (x86)\Google\Chrome\Application" "C:\Program Files\Google\Chrome\Application"
+
+    ./node_modules/.bin/webdriver-manager update --versions.chrome=85.0.4183.38
   displayName: 'add chrome driver to node directory'
```

Comment:
can you briefly explain what the bug was and how these lines fix it, for my own benefit mostly